internal/kubernetes/client.go

```diff
@@ -20,13 +20,26 @@ import (
 	"context"
 	"errors"
 
+	e "github.com/hacbs-contract/ec-cli/pkg/error"
```

Comment:
Wrong import order?

Reply:
Yeah, I didn't _gci_ this, I was thinking of adding gci call to the linter, perhaps I'll do that on this PR. The workaround would be running `make lint-fix` twice.